relayer/src/chain/cosmos.rs

```diff
@@ -209,31 +284,120 @@ impl Chain for CosmosSDKChain {
     }
 
     fn trust_threshold(&self) -> TrustThreshold {
-        TrustThreshold::default()
+        // TODO - get it from src config when avail
+        // TODO - Should this be part of the Chain trait?
+        TrustThreshold {
+            numerator: 1,
+            denominator: 3,
+        }
     }
 
     fn unbonding_period(&self) -> Duration {
         // TODO - query chain
-        Duration::from_secs(24 * 7 * 3)
+        Duration::from_secs(24 * 7 * 3 * 3600)
```

Comment:
As discussed, unclear if we should keep these in the chain trait. Just to keep in mind for future work here.

Reply:
Moved to `impl CosmosSDK`